test/templates/loading-spinner/index.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+{% from "../button/component.njk" import button %}
```

Comment:
Where is button being used? We shouldn't be mixing components in the playground.